app/helpers/users_helper.rb

```diff
@@ -160,36 +157,37 @@ def authors_header(collection, what = "People")
 
   def log_item_action_name(action)
     if action == ArchiveConfig.ACTION_ACTIVATE
-      t('users_helper.log_validated', :default => 'Account Validated')
+      t('users_helper.log_validated', default: 'Account Validated')
+    elsif action == ArchiveConfig.ACTION_CHECK
```

Comment:
This should be changed to `ACTION_TROUBLESHOOT` and `Account Troubleshot`. (Yep, "troubleshot" is what the dictionary says is the past tense: http://www.merriam-webster.com/dictionary/troubleshoot) Also, I think it should go last -- the other items are arranged by the action number, like in the config file.